package.json

```
@@ -62,6 +62,7 @@
     "@bbc/psammead-caption": "^0.1.0",
     "@bbc/psammead-copyright": "^0.2.0",
     "@bbc/psammead-headings": "^0.1.2",
+    "@bbc/psammead-image": "^0.1.0",
```

If you pull in `^0.2.0`, the prop error in the console should disappear. 
```suggestion
    "@bbc/psammead-image": "^0.2.0",
```